infra/monitor/prometheus/prometheus.yml

```
@@ -0,0 +1,15 @@
+
+rule_files:
+  - "/etc/prometheus/rules.yml"
+
+scrape_configs:
+  - job_name: "redis"
+    static_configs:
+      - targets: [ "redis_exporter:9121" ]
+  - job_name: "django"
+    metrics_path: /metrics
+    static_configs:
+      - targets: [ "host.docker.internal:8000" ]
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)